jujugui/static/gui/src/app/components/shared/status/application-list/application-list.js

```diff
@@ -5,11 +5,12 @@ const PropTypes = require('prop-types');
 const React = require('react');
 const { urls } = require('jaaslib');
 
+const propTypes = require('../../../../maraca/prop-types');
```

Comment:
It's a little confusing as this isn't the actual `propTypes`, maybe a rename to `maracaPropTypes` or something?